stock_valued_picking_report/report/stock_picking_valued_report.xml

```diff
@@ -13,7 +13,8 @@
                 <th class="text-right"><strong>Taxes</strong></th>
             </t>
         </xpath>
-        <xpath expr="//span[@t-field='pack_operation.product_uom_id']/.." position="after">
+        <xpath expr="//span[@t-field='pack_operation.qty_done_uom_ordered']/."
```

Comment:
Don't work
![captura de pantalla_2017-10-20_09-45-32](https://user-images.githubusercontent.com/7701001/31810454-2d1afaac-b57c-11e7-9341-65999e3eab04.png)
![captura de pantalla_2017-10-20_09-44-06](https://user-images.githubusercontent.com/7701001/31810458-30ff6d60-b57c-11e7-80c9-7d4df4c87e0c.png)

Reply:
As we said yesterday, the delivery note should be able to print before validating.
Why do you remove a dot?